model-archiver/model_archiver/model_packaging.py

```diff
@@ -36,7 +37,7 @@ def package_model(args, manifest):
 
         # Step 2 : Zip 'em all up
         ModelExportUtils.archive(export_file_path, model_name, model_path, manifest, args.archive_format)
-
+        shutil.rmtree(model_path)
```

Comment:
Not sure if we need this. [copy_artifacts](https://github.com/pytorch/serve/blob/master/model-archiver/model_archiver/model_packaging_utils.py#L136) already cleans up the temp files.

[Test failures](https://serverlessrepo-github-codebuild-logs-cp-buildlogs-1hphnohzwif68.s3.amazonaws.com/cf0da0bd-2f14-42a6-8737-5939114e8deb/build.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=ASIAZFY6OBP56BP636PD%2F20210323%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20210323T194453Z&X-Amz-Expires=600&X-Amz-SignedHeaders=host&X-Amz-Security-Token=IQoJb3JpZ2luX2VjEIz%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FwEaCXVzLWVhc3QtMSJIMEYCIQCIc9UgDsaCLDLIk8uLcEsW4i%2F2CRURtsYbbIMbbQ%2F0lwIhAJblyPaGY9uQME4Dbjz%2Fgn1q3i4qVvYgURC7p9a3Hi5mKoYCCMX%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FwEQARoMNjMwODg3MTU2NzMxIgx0QDA%2FfDB62lC%2FMREq2gGcUPTblRlMLQ34LQtWz3baJne3YQ3kv%2Fs7uez1m1BXS0NR2roGR7%2FZonsS9L5POLk6SuExpNYE6MxDcoLMuiSg4Nc3F1Xb3kp2UJhq5joN%2Blc9ejamFJZ9F3XVJ462D8g7wpLUrXy3AbIEcTtHdUQ8LmX0E5SmaeAaiA%2F9jjmunz8ll1atT4FzkJSRu05PgYbQefvSLMeGccDRHBYCfvVBFLlfdMt9WQZhWWxkmAOCiMQCaLK9X3MRLoCzpUXl0JvL9t6fwQXwmON91kdW5XxJweMXA45dijL2PTC0iumCBjrfATVicbutDeo5WZIK2T%2FhBXY1BvLtvpfsZM7r014c0BTCczes2i%2FUTlOSNW6N3lSm1KXDv8OepR%2BqvFmxeoypuLglFwN58Dhz2VGGzaRVtaamsASStL97qjYk2udJe6O1cS4RI2L6N9vH7Nb1KEXBXN%2FT8mIW6zjOVsLsUY%2FAs0mzJgUAai8GmeX5M3YyX8YlU10lsEPnucJHCeff1pvN3uScHbLuekk7%2FQcfsrIysqnxQT3wjAkPiq1bKSdli1724fWlbtnNGiwZVknPxZdfbcxsrBLrVkBKg4hU0JyP8U0%3D&X-Amz-Signature=cd5708b00230ae9344c85034e693b510d8352b59925c30b83fa46b2d2e33f255)

Reply:
@maaquib as per our offline chat, we are removing the model_path here if it is existing and right after that we are making the directory.